packages/devops/scripts/deployment-vm-app/install.sh

```diff
@@ -32,6 +32,9 @@ git reset --hard origin/${GIT_BRANCH}
 # Yarn install
 yarn install --frozen-lockfile
 
+# TODO: 1. Wrong password in lastpass to login lesmis@tupaia.org
+#       2. It will pull wrong key for REACT_APP_LESMIS_API_URL in lesmis .env
+#       3. change all service api urls to localhost as they interact locally
```

Comment:
Good idea, let's leave it for future when we can do it for both our infra and the LESMIS infra at the same time